src/context.cpp

```diff
@@ -122,15 +123,51 @@ Context::Context(
     // TODO: What is this and how do I un-Arduino it?
     analogSetAttenuation(ADC_11db);
 
-    // Set the moisture check frequency to a default of 1 hour
-    // TODO: get minute_moisture_check_freq from and write to persistent memory, flash?
-    minute_soil_moisture_check_freq = 60;
+    // TODO: delay all storage access until after storage_init,
+    //       or call storage_init if it has not already been called from within here
```

Comment:
This is done, remove this TODO.

Reply:
Addressed in https://github.com/jCallon/squirt/pull/3/commits/09245cdc3c711cc6a98ca41015751c44460d8aab.